index.html

```diff
@@ -0,0 +1,147 @@
+            
+        </div>
+        <div class="container" id="hamburger">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+        </div>
+        <ul id="myLinks">
+            <li>
+                <a href="index.html">Home</a>
+            </li>
+            <li>
+                <a href="about.html">About Us</a>
+            </li>
+        </ul>
+    </nav>
+    <section class="intro">
+        <!-- <img class="intro-img" src="./assets/resto1.jpg" alt="intro-img"> -->
+        <div class="intro-content">
+            <h2 class="intro-message">"Hello you there come and eat"</h2>
+            <h1>Welcome to Isekai Restaurant</h1>
+        </div> 
+        <div class="intro-about-desc">
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
+                Deleniti ea accusamus odit. Praesentium ullam deleniti reprehenderit 
+                ea mollitia explicabo blanditiis corporis quaerat? Harum eos quam 
+                voluptas explicabo vero nihil fuga?
+            </p>
+        </div>
+        <h2 class="intro-text-date"> 2015.10.15(THU)~16(FRI)</h2>
+    </section>
+    <section class="news-events" id="news">
+        <h2>News & Events</h2>
+        <div class="line"></div>
+        <div class="news col-12 col-lg-10 d-md-flex justify-content-center">
+            <div class="card col-10 mx-auto col-md-3 m-3 d-flex d-md-block flex-row" >
+                <img src="./assets/steak.jpeg" class="card-img-top" alt="steak">
+                <div class="card-body w-100">
+                    <h5 class="card-title">News</h5>
+                    <p class="card-text">Minima aperiam ullam veniam fugiat asperiores sed qui, voluptatum eius? Quisquam </p>
+                    <!-- <a href="#" class="btn btn-primary">Go somewhere</a> -->
+                </div>
+            </div>
+            <div class="card col-10 mx-auto col-md-3 m-3 d-flex d-md-block flex-row" >
+                <img src="./assets/resto3.jpg" class="card-img-top" alt="resto">
+                <div class="card-body w-100">
+                    <h5 class="card-title">News</h5>
+                    <p class="card-text"> Minima aperiam ullam veniam fugiat asperiores sed qui, voluptatum eius? Quisquamat!</p>
+                    <!-- <a href="#" class="btn btn-primary">Go somewhere</a> -->
+                </div>
+            </div>
+            <div class="card col-10 mx-auto col-md-3 m-3 d-flex d-md-block flex-row" >
+                <img src="./assets/decert-cup.jpg   " class="card-img-top" alt="decert">
+                <div class="card-body w-100">
+                    <h5 class="card-title">Event</h5>
+                    <p class="card-text"> Minima aperiam ullam veniam fugiat asperiores sed qui, voluptatum eius? Quisquam </p>
+                    <!-- <a href="#" class="btn btn-primary">Go somewhere</a> -->
+                </div>
+            </div>
+        </div>
```

Comment:
I also mentioned this earlier but it was not addressed

- [ ] You're expected to have 5 articles here in desktop view but you currently have 3. Kindly add 2 more

Yours | Orignal
----- | -----
![image](https://user-images.githubusercontent.com/16058470/124146605-d7258d00-da85-11eb-84ba-55928c54b1f8.png) | ![image](https://user-images.githubusercontent.com/16058470/124146669-e6a4d600-da85-11eb-8c3f-f410bc7045a1.png)

Reply:
done